dpnp/dpnp_array.py

```diff
@@ -915,10 +915,14 @@ def max(
 
         return dpnp.max(self, axis, out, keepdims, initial, where)
 
-    def mean(self, axis=None, **kwargs):
-        """Returns the average of the array elements."""
+    def mean(self, axis=None, dtype=None, keepdims=False, out=None, where=True):
```

Comment:
to align with numpy:
```suggestion
    def mean(self, axis=None, dtype=None, out=None, keepdims=False, *, where=True):
```